experiments/integrated/fluxnet/snow_soil/simulation.jl

```diff
@@ -0,0 +1,300 @@
+    earth_param_set = earth_param_set,
+);
+snow_args = (; domain = snow_domain, parameters = snow_parameters);
+snow_model_type = Snow.SnowModel
+land_input = (
+    atmos = atmos,
+    radiation = radiation,
+    runoff = ClimaLand.Soil.NoRunoff(),#SiteLevelSurfaceRunoff(),
+)
+land = ClimaLand.LandHydrologyModel{FT}(;
+    land_args = land_input,
+    soil_model_type = soil_model_type,
+    soil_args = soil_args,
+    snow_args = snow_args,
+    snow_model_type = snow_model_type,
+)
+Y, p, cds = initialize(land)
+exp_tendency! = make_exp_tendency(land)
+imp_tendency! = make_imp_tendency(land)
+jacobian! = ClimaLand.make_jacobian(land);
+jac_kwargs = (; jac_prototype = ImplicitEquationJacobian(Y), Wfact = jacobian!);
+set_initial_cache! = make_set_initial_cache(land)
+
+
+#Initial conditions
+Y.soil.ϑ_l =
+    drivers.SWC.status != absent ?
+    drivers.SWC.values[1 + Int(round(t0 / DATA_DT))] : soil_ν / 2
+Y.soil.θ_i = 0
+T_0 =
+    drivers.TS.status != absent ?
+    drivers.TS.values[1 + Int(round(t0 / DATA_DT))] :
+    drivers.TA.values[1 + Int(round(t0 / DATA_DT))]
+ρc_s =
+    volumetric_heat_capacity.(
+        Y.soil.ϑ_l,
+        Y.soil.θ_i,
+        soil_ps.ρc_ds,
+        earth_param_set,
+    )
+Y.soil.ρe_int =
+    volumetric_internal_energy.(Y.soil.θ_i, ρc_s, T_0, earth_param_set)
+
+Y.snow.S .= 0.0
+Y.snow.U .= 0.0
+set_initial_cache!(p, Y, t0)
+
+saveat = Array(t0:dt:tf)
+sv = (;
+    t = Array{Float64}(undef, length(saveat)),
+    saveval = Array{NamedTuple}(undef, length(saveat)),
+)
+saving_cb = ClimaLand.NonInterpSavingCallback(sv, saveat)
+
+updateat = deepcopy(saveat)
+model_drivers = ClimaLand.get_drivers(land)
+updatefunc = ClimaLand.make_update_drivers(model_drivers)
+driver_cb = ClimaLand.DriverUpdateCallback(updateat, updatefunc)
+cb = SciMLBase.CallbackSet(driver_cb, saving_cb)
+# TIME STEPPING
+timestepper = CTS.ARS111();
+ode_algo = CTS.IMEXAlgorithm(
+    timestepper,
+    CTS.NewtonsMethod(
+        max_iters = 3,
+        update_j = CTS.UpdateEvery(CTS.NewNewtonIteration),
+    ),
+);
+
+# Problem definition and callbacks
+prob = SciMLBase.ODEProblem(
+    CTS.ClimaODEFunction(
+        T_exp! = exp_tendency!,
+        T_imp! = SciMLBase.ODEFunction(imp_tendency!; jac_kwargs...),
+        dss! = ClimaLand.dss!,
+    ),
+    Y,
+    (t0, tf),
+    p,
+);
+sol = SciMLBase.solve(prob, ode_algo; callback = cb, dt = dt, saveat = t0:dt:tf);
+
+# Plotting
+daily = sol.t ./ 3600 ./ 24
+savedir = joinpath(climaland_dir, "experiments/integrated/fluxnet/snow_soil")
+
+
+# Water content
+
+fig = Figure(size = (1600, 1200), fontsize = 26)
+ax1 = Axis(fig[2, 2], ylabel = "SWC", xlabel = "Days")
+lines!(
+    ax1,
+    daily,
+    [parent(sol.u[k].soil.ϑ_l)[end] for k in 1:1:length(sol.t)],
+    label = "2cm",
+)
+lines!(
+    ax1,
+    daily,
+    [parent(sol.u[k].soil.θ_i)[end] for k in 1:1:length(sol.t)],
+    label = "2cm, ice",
+)
+
+lines!(ax1, seconds ./ 3600 ./ 24, drivers.SWC.values[:], label = "Data, ?cm")
```

Comment:
`?cm`?

Reply:
I cant find a reported depth where the sensor is...hence "? cm" :(

Reply:
I changed label to "Unknown depth"